src/ft/mod.rs

```diff
@@ -346,87 +448,6 @@ impl FreeTypeRasterizer {
         Ok(glyph.font_key)
     }
 
-    fn get_rendered_glyph(&mut self, glyph_key: GlyphKey) -> Result<RasterizedGlyph, Error> {
```

Comment:
There seems to be no reason for this to be a separate method, so I decided to just inline it.